cli-feed-v4.json

```diff
@@ -41,7 +41,7 @@
       "hidden": false
     },
     "v4": {
-      "release": "4.24.0",
+      "release": "4.26.0",
```

Comment:
This would override the changes done here
https://github.com/Azure/azure-functions-tooling-feed/pull/395

Reply:
Yea, I will rebase my changes on top of this.